pages/about.tsx

```diff
@@ -1,31 +1,61 @@
 import Head from 'next/head';
+import Header from '../components/Header';
 
-const AboutHead = () => (
-  <Head>
-    <title>About - Jupiter</title>
-    <meta name="description" content="Generated by create next app" />
-    <link rel="icon" href="/favicon.ico" />
-  </Head>
-);
-
-export default function About() {
+const About = () => {
   return (
     <>
-    <AboutHead/>
-    <div className="p-5">
-      <div className="mx-auto max-w-2xl">
-        <h1 className="text-lg font-bold py-4">About Jupiter</h1>
-        {[
-          'Incidunt quam amet ullam nesciunt assumenda occaecati. Animi quibusdam impedit alias quam omnis. Autem nisi minima iusto sunt. Repellat dignissimos sit aperiam nesciunt autem temporibus id et. Ut quo consectetur illo in possimus et non quas.',
-          'Neque quis dolores minima quaerat est eligendi recusandae. Aut voluptatem quos consectetur excepturi repellat dolores. Nam sapiente voluptatem eum laboriosam possimus omnis. Veniam atque et voluptatem aut.',
-          'Sequi rerum quisquam recusandae. Necessitatibus incidunt debitis doloremque error qui doloribus optio sunt. Est ea eligendi fuga minima facere voluptatem omnis. Iure ut et quo qui eaque. Dolor et corporis tenetur rerum doloremque non ea. Saepe voluptatem suscipit ex.',
-        ].map((pText, i) => (
-          <p key={i} className="mb-4">
-            {pText}
-          </p>
-        ))}
-      </div>
-    </div>
+      <Head>
+        <title>Jupiter</title>
+        <meta name="description" content="Generated by create next app" />
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
+      <main className="md:pl-72">
+        <Header />
+        <div className="w-11/12 mx-auto">
+          <h1 className="text-2xl font-medium text-slate-500 pl-10 pt-10">
+            About us
+          </h1>
+          <div className="w-full bg-slate-100 mt-10 rounded-lg p-8">
+            <h1 className="text-2xl font-medium text-slate-500">
+              Project Jupiter
+            </h1>
+            <h3 className="font-light text-slate-500 mt-4">
+              Project description
```

Comment:
Visually, do we have a need for "Project Description" to be here? It seems redundant